packages/accounts-controller/src/AccountsController.ts

```diff
@@ -836,6 +837,7 @@ export class AccountsController extends BaseController<
           patch.added.push({
             address,
             type: keyring.type,
+            entropySource: keyring.metadata.id,
```

Comment:
WDYT about having a reference to the `keyring` itself here, so that we can re-use this later to inject the `entropySource` only for HD keyrings?

Reply:
we already have `type: keyring.type,` so I used that instead